_posts/2020/2020-10-19-3-tipps-for-feedback-in-a-Scrum-Team

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: 3 Tipps for feedback in a Scrum Team
+date: 2020-10-15
+header_image: 
+header_position: center
+header_overlay: true
+category: methods-and-tools
+tags: ["feedback", "Agile", "Scrum"]
+authors: ["Stefanie"]
+about_authors: ["sholller"]
+---
+Uhhh, yet another blog post about giving feedback… 
+There are already thousands of articles about this topic. 
+Now that I start writing this blog post I feel really reluctant to do this, since you have probably heard all of this before and nothing I write will be new to you. 
+But I’d like to invite you to ask yourself: When was the last time someone annoyed and you directly went to this colleague afterwards to tell him how this made your feel?
+Can’t recall when this happened? Then please go on reading here!
+When our teammates do something that makes us angry there are typically 5 reactions:
+1. This was so bad, he/she should realize this on her/his own
+2. He/she needs to know this was bad, but the Scrum Master has to tell him/her
+3. Can’t the manager say this is unacceptable behaviour?
+4. I’ll wait for the next retrospective/annual feedback to bring this up
+5. I have said this so often, he/she should know by now.
+
+## Honest Feedback means you care
+
+No, your colleague will probably not guess what you think is unacceptable. 
+Norms is a team are formed by conversation about it. 
+They are the common denominator of all team members expectations of a wanted behaviour. Therefore: if no one calls out unacceptable behaviour it will feel like accepted behaviour over time. 
+Even if it does feel uncomfortable, giving constructive feedback about things you did not like, helps your colleagues to get better. 
+Telling them means you care about them, that you value them and that you want them to improve. 
+The awkward feeling of saying: “Hey, I think you could do better here” quickly disappears if you say: “How do you see it? Is there a way I can help you to improve?” 
+This is even more important if you are working remotely. 
+I often hear people say: “I don’t think it’s appropriate to discuss this remote. I prefer to speak about it face to face.” 
+It’s ok if you wait for your colleague to be back in the office the next day, but don’t postpone it for longer. 
+With colleagues, that are working from home or in different locations I often witness, that the feedback will never be given. 
+And I know from colleagues that work remotely, that they appreciate very much if someone reaches out to them and asks for a call. 
+
```

Comment:
The great art of giving valuable feedback is to leave out personal interpretations and focus only on the perception and effect of the behavior. In addition, I-messages are essential, since it is all about your perception.

A simple and practicable way of giving constructive feedback in an appreciative and respectful manner comes from Rosenberg's Nonviolent Communication and is divided into three sections.

**1. Perception:** What have I observed?
**2. Effect:** What was the effect on me?
**3. Wish:** What is my wish for the future?

For example: 
**1. Perception:** "I noticed that you seemed absent at our last meeting. You often looked at your mobile phone and hardly participated in the joint discussion." 
**2. Effect:** "I felt that another topic was on your mind at that moment and that you were absent. In this way you miss the contents of our meeting and do not contribute anything from your side. That's a pity, because we are running the project together."
**3. Wish:** "My wish for the future is that you simply seek the conversation with me in advance. If you have an urgent topic on your mind, it makes sense to clarify it first and then be present at the meeting with a clear head. With pleasure we can then also simply move the appointment. Just talk to me openly."

These three steps are a good guideline that I often use before I give feedback.
But now let's move on to the 3 tipps for feedback in a Scum-Team.

Reply:
Added to the article